.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,38 @@
+## FreeCodeCamp - Wiki Pull Request Template
+
+Make sure your contribution addresses the points mentioned below:
+
+**Filenames:**
+
+- Your filename shoud NOT contain any special characters or spaces.
+- Use only "-" to separate words in the filename.
+- Filenames should have extention ".md"
+
+**Formatting:**
+
+- Ensure that you use markdown syntax for articles.
+- Checkout the guides and cheetsheets for reference:
```

Comment:
oops yup, actually we removed the cheat sheets in favor of offical github docs, I'll remove it.